apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageController.java

```diff
@@ -78,6 +83,13 @@ public Page<PageSummary> search(
         return results;
     }
     
+    @PostMapping("/add")
+    public PageCreateResponse createPage(@RequestBody PageCreateRequest request) {
+    	Long userId = userDetailsProvider.getCurrentUserDetails().getId();
+    	return creator.createPage(request,userId);
+    }
```

Comment:
Hmm looks like I painted you into a corner for the mapping. The @PostMapping above this one that is used for search should really be updated to `@PostMapping("/search")` so that this one could be `@PostMapping`.